
#uint8
float64 target_time

---

bool result
